pulsar/consumer_test.go

```diff
@@ -4691,3 +4691,66 @@ func TestPartitionConsumerGetLastMessageIDs(t *testing.T) {
 	}
 
 }
+
+func TestLookupConsumer(t *testing.T) {
+	client, err := NewClient(ClientOptions{
```

Comment:
What does this test try to verify? It's a trivial end-to-end test. The message properties are never related to the lookup properties.

Reply:
Since the lookup properties rely on the customized lookup logic, I think it's okay to not add any test.

Reply:
Yes, I agree with you, this is a basic functional test of duplication to ensure that the  `LookupProperties` do not affect the existing functionality.

Reply:
If so, we can just configure values in `ProducerMessage`. Configuring key and properties might make users think if lookup properties will affect them.